src/drawer/components/EmptyScreen/EmptyScreen.tsx

```diff
@@ -10,12 +11,12 @@ import {
 } from './EmptyScreen.style';
 
 interface EmptyScreenProps {
-  boldText: string;
-  subText: string;
   type: 'SEARCH' | 'STAR' | 'MYDRAWER' | 'PROVIDER';
 }
 
-export const EmptyScreen = ({ boldText, subText, type }: EmptyScreenProps) => {
+export const EmptyScreen = ({ type }: EmptyScreenProps) => {
+  const { boldText, subText } = { ...NOT_FOUND_TEXT[type] };
```

Comment:
```suggestion
  const { boldText, subText } = NOT_FOUND_TEXT[type];
```
spread 안써두 될 것 같아서,,

Reply:
헉 ! 당장반영